documentation/Color system.md

```diff
@@ -15,25 +15,25 @@
-| <pre>--p-backdrop</pre>                  | For use as the background color of the backdrop component for navigation and modal. This color has an alpha of `0.5`.                 | ![](https://www.gifpng.com/64x64/000000/FFFFFF?border-width=16&border-type=rectangle&border-color=fafafa&text=%20)<br />#000000 | ![](https://www.gifpng.com/64x64/000000/FFFFFF?border-width=16&border-type=rectangle&border-color=0c0d0e&text=%20)<br />#000000 |
-| <pre>--p-overlay</pre>                   | For use as the background color of elements which lay on top of surfaces to obscure their contents. This color has an alpha of `0.5`. | ![](https://www.gifpng.com/64x64/ffffff/FFFFFF?border-width=16&border-type=rectangle&border-color=fafafa&text=%20)<br />#ffffff | ![](https://www.gifpng.com/64x64/212121/FFFFFF?border-width=16&border-type=rectangle&border-color=0c0d0e&text=%20)<br />#212121 |
-| <pre>--p-shadow-from-ambient-light</pre> | For use in building shadows for popovers, cards, and modals. This color has an alpha of `0.05`.                                       | ![](https://www.gifpng.com/64x64/161717/FFFFFF?border-width=16&border-type=rectangle&border-color=fafafa&text=%20)<br />#161717 | ![](https://www.gifpng.com/64x64/161717/FFFFFF?border-width=16&border-type=rectangle&border-color=0c0d0e&text=%20)<br />#161717 |
-| <pre>--p-shadow-from-direct-light</pre>  | For use in building shadows for popovers, cards, and modals. This color has an alpha of `0.15`.                                       | ![](https://www.gifpng.com/64x64/000000/FFFFFF?border-width=16&border-type=rectangle&border-color=fafafa&text=%20)<br />#000000 | ![](https://www.gifpng.com/64x64/ffffff/FFFFFF?border-width=16&border-type=rectangle&border-color=0c0d0e&text=%20)<br />#ffffff |
-| <pre>--p-hint-from-direct-light</pre>    | For use in building shadows scrollables.                                                                                              | ![](https://www.gifpng.com/64x64/000000/FFFFFF?border-width=16&border-type=rectangle&border-color=fafafa&text=%20)<br />#000000 | ![](https://www.gifpng.com/64x64/bababa/FFFFFF?border-width=16&border-type=rectangle&border-color=0c0d0e&text=%20)<br />#bababa |
+|CSS variable|Description|Light mode|Dark mode|
+|---|---|---|---|
+|<pre>--p-background</pre>|For use as a background color, in components such as Page and Frame backgrounds. |![](https://www.gifpng.com/64x64/fafafa/FFFFFF?border-width=16&border-type=rectangle&border-color=fafafa&text=%20)<br />#fafafa|![](https://www.gifpng.com/64x64/0b0c0d/FFFFFF?border-width=16&border-type=rectangle&border-color=0b0c0d&text=%20)<br />#0b0c0d|
```

Comment:
Looks like all of the changes to output values are the result of rounding RGB values in the color factory. If you look at the rich diff you’ll see the are values off by 1 and visually indistinguishable. We could potentially remove this and see if Theo handles floating point numbers.

Reply:
I'm comfortable with the changes as-is. We can always iterate on the color factory for later releases if these are not an accurate representation of intended color variants.